lib/netrc.c

```diff
@@ -81,8 +97,20 @@ static int parsenetrc(const char *host,
   file = fopen(netrcfile, FOPEN_READTEXT);
   if(file) {
     bool done = FALSE;
-    char netrcbuffer[4096];
-    int  netrcbuffsize = (int)sizeof(netrcbuffer);
+    char *netrcbuffer = NULL;
+    int  netrcbuffsize;
+
+    netrcbuffsize = get_file_sz(file);
+    if(netrcbuffsize++ == -1) {
+      retcode = NETRC_FAILED;
+      goto out;
+    }
```

Comment:
Without this change, a *super huge* .netrc file was a nuisance and slow. With this change, you introduce a super huge alloc as well. I fear this can have some undesired consequences. **If** we should allocate memory for the entire file, then I think it needs to have a capped maximum size, like maybe 8MB

Reply:
ACK. I will close this PR then, because at that point it'd be only a somewhat endless race.